tests/chainer_tests/functions_tests/loss_tests/test_discriminative_margin_based_clustering_loss.py

```diff
@@ -17,48 +17,50 @@
     'beta': [1],
     'gamma': [0.001],
     'norm': [1],
-    'result_l_dist': [6.0776708],
```

Comment:
How about keeping the original test as well? We need to make sure that this loss is backwards-compatible with the previous version.

Reply:
Ah, as we change the calculation method, the values also have to be adjusted for tests not to fail.